docs/csharp/quick-starts/branches-and-loops.yml

```diff
@@ -72,7 +73,7 @@ items:
     > The indentation under the `if` and `else` statements is for human readers.
     > The C# language doesn't treat indentation or whitespace as significant. 
     > The statement following the `if` or `else` keyword will be executed based
-    > on the condition. All the samples in this tutorial follow a common
+    > on the condition. All the samples in this quick start follow a common
     > practice to indent lines based on execution.
```

Comment:
same comment as earlier about "based on execution"

Reply:
I tried the same change here.